code/src/index.css

```diff
@@ -1,13 +1,153 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
+  background-color: black;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+main {
+  /* max-width: 1200px; */
+  margin: 0 auto;
+  width: 100%;
+  /* padding: 0 50px; */
+}
+
+.movie-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.movie-info {
+  display: none;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.75);
+  padding: 20px;
+}
+
+.movie-list a:hover .movie-info {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.movie-card {
+  width: 25%;
+  position: relative;
+  color: #fff;
+  text-decoration: none;
+}
+
+.movie-list a img {
+  width: 100%;
+}
+
+.detail-page {
+  position: relative;
+}
+
+.backLink svg {
+  width: 50px;
+}
+
+.backLink {
+  color: #fff;
+  text-decoration: none;
+  display: inline-flex;
+  align-items: center;
+  font-size: 20px;
+  font-weight: bold;
+  position: absolute;
+  top: 5%;
+  left: 5%;
+}
+
+.detail-page a:hover {
+  transform: scale(1.1);
+}
+
+.backdrop {
+  min-height: 100vh;
+  background-size: cover;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+}
+
+.summary {
+  display: flex;
+  color: #fff;
+  padding: 50px;
+}
+
+.summary img {
+  border: 2px solid white;
+  margin-left: 1em;
+  max-height: 517px;
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin-left: 40px;
+  margin-bottom: 40px;
+}
+
+.details-header {
+  font-size: 28px;
+}
+
+.details-header span {
+  font-size: 18px;
+  color: red;
+  margin-left: 15px;
+}
+
+p {
+  max-width: 400px;
+  font-size: 18px;
+}
+
+.not-found-container,
+.error-message {
+  display: grid;
+  color: #fff;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@media screen and (max-width: 720px) {
```

Comment:
You've done a great job of styling for responsiveness, looks really nice on mobile too.